.github/workflows/build-test.yaml

```diff
@@ -54,6 +54,29 @@ jobs:
           path: coverage
           retention-days: 5
 
+  playwright:
+    needs: [yarn_cache]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+          cache: 'yarn'
+      - run: yarn install --frozen-lockfile
+      - run: yarn start &
+      - run: yarn install --frozen-lockfile
+        working-directory: playwright
+      - run: npx playwright install --with-deps
+        working-directory: playwright
```

Comment:
Note, after yarn install playwright needs more steps to start working https://playwright.dev/docs/cli#install-system-dependencies